### PR TITLE
Updated wording of particle types (ParticleEmitterEditor.js)

### PR DESCRIPTION
**Pull request by tristanbob**
Updated wording of particle types.

### CODE REVIEW

**Review by arthuro555 — APPROVED** [review]
👍

File: newIDE/app/src/ObjectEditor/Editors/ParticleEmitterEditor.js [hunk 1]
```diff
@@ -57,15 +57,15 @@ export default class ParticleEmitterEditor extends React.Component<
         >
           <SelectOption
             value={gd.ParticleEmitterObject.Point}
-            primaryText={t`Point`}
+            primaryText={t`Circle`}
           />
           <SelectOption
             value={gd.ParticleEmitterObject.Line}
-            primaryText={t`Line`}
+            primaryText={t`Rectangle`}
```
Comment (Bouh) [inline newIDE/app/src/ObjectEditor/Editors/ParticleEmitterEditor.js:64]: Humm no, all settings below this select are for a line, like **length** and **thickness**.
If we rename it rectangle you have to edit the parameters below for width and height, I recommend going back to **line**!

Comment (4ian) [reply newIDE/app/src/ObjectEditor/Editors/ParticleEmitterEditor.js:64]: This is very true, I was a bit too fast. I'm changing this back to line. 
Thanks for your careful review!

Comment (4ian) [reply newIDE/app/src/ObjectEditor/Editors/ParticleEmitterEditor.js:64]: Done in aeecb0e29f5a14f3e199a99268c57bd519cff5ce

**Pull request merged**